queue.c

```diff
@@ -1851,6 +1851,14 @@ BaseType_t xQueueSemaphoreTake( QueueHandle_t xQueue,
                              * again, but only as low as the next highest priority
                              * task that is waiting for the same mutex. */
                             uxHighestWaitingPriority = prvGetDisinheritPriorityAfterTimeout( pxQueue );
+
+                            /* uxHighestWaitingPriority returned by prvGetDisinheritPriorityAfterTimeout
+                             * is set to ( configMAX_PRIORITIES - xItemValue ). The implementation ensures
+                             * that the xItemValue is set to ( configMAX_PRIORITIES - uxPriority ). The maximum
+                             * value of uxPriority of a task is ( configMAX_PRIORITIES - 1 ). Therefore,
+                             * uxHighestWaitingPriority is smaller than configMAX_PRIORITIES. The overrun
+                             * warning is false positive. */
+                            /* coverity[overrun] */
```

Comment:
Is this a valid coverity suppression?

Reply:
Yes, this is a valid coverity suppression.